<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirmwareUpgrade</class>
 <widget class="QDialog" name="FirmwareUpgrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmware Upgrade</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Firmare file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectFirmware">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkResetSettings">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset settings</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFirmwareFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpgrade">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
